### ISSUE TITLE
Check SatelliteDataSource and NWPDataSource put x and y dims in the same order

### ISSUE THREAD

**Issue by JackKelly** [openclimatefix/nowcasting_dataset#411]
I have a feeling that NWP puts the dims in this order: y, x.  Whilst satellite puts the dims in order x, y.

**Comment by peterdudfield** [comment]
I agree should be standard way, luckily in dataloader we re-order them anyway - just incase

see [here](https://github.com/openclimatefix/nowcasting_dataloader/blob/main/nowcasting_dataloader/data_sources/nwp/nwp_model.py#L126) for in nwp and [here](https://github.com/openclimatefix/nowcasting_dataloader/blob/main/nowcasting_dataloader/xr_utils.py#L91) for function

**Comment by JackKelly** [comment]
Love it!

So, I guess I can close this issue?

Or do we still want NWPDataSource to put its dims into the order x, y?
